.buildbot.sh

```diff
@@ -27,7 +28,12 @@ ulimit -d $((1024 * 1024 * 10)) # 10 GiB
 
 # Build extended tools and install into TARBALL_TOPDIR.
 mkdir -p ${TARBALL_TOPDIR}
-/usr/bin/time -v ./x.py install --config .buildbot.config.toml
+# We should be able to do this, but `x.py install` is broken.
```

Comment:
What does "this" refer to here? [I eventually guessed, but it's a bit ambiguous.]

Reply:
The following commented line. We can add some whitespace to make it more obvious?

Reply:
It'll still be ambiguous I suspect: I think the comment needs rephrasing.

Reply:
Give 5057cab a go.